README.md

```diff
@@ -301,26 +338,28 @@ It comes with a few more options as well:
 | blacklist | An array of parameter key names to redact from logging, in path.to.key format | `[]` |
 | redacted_string | The string to use for redacted parameters. | `REDACTED` |
 
-It's important to maintain a safe blacklist should you decide to log parameters; gruf does no 
-parameter sanitization on its own. We also recommend blacklisting parameters that may contain 
+It's important to maintain a safe blacklist should you decide to log parameters; gruf does no
+parameter sanitization on its own. We also recommend blacklisting parameters that may contain
 very large values (such as binary or json data).
 
 ## Plugins
 
 You can build your own hooks and middleware for gruf; here's a list of known open source gems for
 gruf that you can use today:
- 
+
 * [gruf-zipkin](https://github.com/bigcommerce/gruf-zipkin) - Provides a [Zipkin](https://zipkin.io)
 integration for gruf
 * [gruf-circuit-breaker](https://github.com/bigcommerce/gruf-circuit-breaker) - Provides circuit breaker
 support for gruf services
-* [gruf-profiler](https://github.com/bigcommerce/gruf-profiler) - Profiles and provides memory usage 
+* [gruf-profiler](https://github.com/bigcommerce/gruf-profiler) - Profiles and provides memory usage
 reports for gruf services
 
 ## Demo Rails App
 
-There is a [demonstration Rails application here](https://github.com/bigcommerce/gruf-demo) you can 
-view and clone that shows how to integrate Gruf into an existing Rails application. 
+There is a [demonstration Rails application here](https://github.com/bigcommerce/gruf-demo) you can
+view and clone that shows how to integrate Gruf into an existing Rails application.
+
+And another demo [Rails_Communicate_Rails](https://github.com/pathbox/Rails_Communicate_Rails) This demo has tow Rails App, one is grpc client, another is grpc server. It is "say hello" example. It is very simple, but it make you know `gruf` quickly.
```

Comment:
Thanks for creating this! Though, I'd rather keep the referenced demo apps in the `gruf` organization (we're looking at moving gruf to its own org in the next year) just so that we can make sure they're up-to-date and kept in line with best practices.

For now, can you remove this? We may create a section in the Wiki later that has "3rd Party examples" and such that would be more appropriate here.